assert-camunda-git-emails/check.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -eu
+
+echo "Looking at Git commits in range $GIT_RANGE"
+
+echo "All found email addresses:"
+git log --pretty=tformat:"%ae%n%ce" "$GIT_RANGE" | sort | uniq
+
+# shellcheck disable=SC2126
+NUM_VIOLATIONS=$(git log --pretty=tformat:"%ae%n%ce" "$GIT_RANGE" | \
```

Comment:
what do you think about
```suggestions
VIOLATIONS=$(git log --pretty=tformat:"%ae%n%ce" "$GIT_RANGE" | \
grep -v "@camunda.com\|@users.noreply.github.com\|noreply@github.com")
NUM_VIOLATIONS=$(echo $VIOLATIONS | wc -l)
UNIQUE_VIOLATIONS=$(echo $VIOLATIONS | sort | uniq)
```
and use those later in the logic, to avoid code duplication?

Reply:
I tried it just now but it gets ugly quickly e.g. when trying to count the number of lines stored in a bash variable (`echo "" | wc -l` yields 1 and `echo "foo" | wc -l` also yields 1).

In my initial proposal I tried to keep this bash script as dumb as possible to avoid nasty edge cases. If it grows more complex, we should migrate to Python. I think the code duplication avoided is not that big (and part will be achieved with having an environment variable with a default list of email addresses).